grabdish/deploy.sh

```diff
@@ -3,7 +3,7 @@
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 
 #MS="frontend-helidon order-helidon supplier-helidon-se inventory-helidon inventory-springboot"
-MS="frontend-helidon order-helidon supplier-helidon-se inventory-helidon"
+MS="frontend-helidon order-helidon supplier-helidon-se inventory-helidon foodwinepairing-python"
```

Comment:
let's take this out of grabdish/deploy.sh and undeploy.sh as we'll make it an additional/explicit step to do that in the workshop alongside the step to set isSuggestiveSaleAIEnabled true in order-helidon and redeploy that.